src/stdlib_sorting.fypp

```diff
@@ -147,14 +151,16 @@ module stdlib_sorting
         max_merge_stack = int( ceiling( log( 2._dp**64 ) / &
                                log(1.6180339887_dp) ) )
 
-    type run_type
+#:for ki, ti, namei in INT_INDEX_TYPES_ALT_NAME
+    type run_type_${namei}$
```

Comment:
Looks perfect @jvdp1, the only comment I have is about the `run_type`. Is this a private derived type? If not, renaming `run_type` to `run_type_default` would introduce a backward incompatible change. So, maybe, they could be `run_type` and `run_type_low` instead?

Reply:
`run_type` is `private` . I renamed it as it was easier to do so with the `fypp` loop.

Reply:
Great - good to go then!